gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyLogging.java

```diff
@@ -94,4 +104,10 @@ private String getVoltageCheckResult(DichotomyDirection direction, Optional<Volt
         }
         return NONE;
     }
+
+    private String getTimestampLocalized(OffsetDateTime timestamp) {
+        OffsetDateTime localTime = java.time.OffsetDateTime.ofInstant(timestamp.toInstant(), zoneId);
+        DateTimeFormatter df = DateTimeFormatter.ofPattern(DATE_TIME_FORMAT);
```

Comment:
You could define a constant instead of recreating a new formatter each time you call the `getTimestampLocalized()` method.

Replace the definition of `DATE_TIME_FORMAT` with something like :
`private static final String DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy'-'MM'-'dd' 'HH':'mm");`
(since you don't need the String `DATE_TIME_FORMAT` anywhere else).